ajax/libs/jQuery-Paging/package.json

```diff
@@ -0,0 +1,38 @@
+{
+  "name": "jQuery-Paging",
+  "title": "jQuery Pagination Plugin",
+  "description": "Probably the most advanced jQuery pagination plugin.",
+  "version": "1.2.0",
+  "keywords": [
+    "paging",
+    "pagination",
+    "pager",
+    "navigation"
+  ],
+  "filename": "jquery.paging.min.js",
+  "author": {
+    "name": "Robert Eisele",
+    "url": "http://www.xarg.org/"
+  },
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/infusion/jQuery-Paging"
+  },
+  "licenses": [
+    "MIT",
+    {
+      "type": "GPLv2",
+      "url": "http://www.opensource.org/licenses/GPL-2.0"
+    }
+  ],
+  "demo": "http://www.xarg.org/2011/09/jquery-pagination-revised/",
+  "npmName": "jQuery-Paging",
```

Comment:
There is no official npm package of the lib. We should use git auto-update config instead of npm. You can take a look at [autoupdate.md](https://github.com/cdnjs/cdnjs/blob/master/documents/autoupdate.md).